pyseir/backtest/backtest.py

```diff
@@ -0,0 +1,432 @@
+import os
+import us
+import matplotlib.backends.backend_pdf
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from datetime import datetime, timedelta
+from matplotlib.dates import DateFormatter
+from multiprocessing import Pool
+from functools import partial
+from pyseir import load_data
+from pyseir.load_data import HospitalizationDataType
+from pyseir.inference.model_fitter import ModelFitter
+from pyseir.utils import get_run_artifact_path, RunArtifact
+from pyseir.backtest.timeseries_metrics import TimeSeriesMetrics, error_type_to_meaning
+
+
+REF_DATE = datetime(year=2020, month=1, day=1)
+
+def load_observations(fips, ref_date=datetime(year=2020, month=1, day=1)):
```

Comment:
This is file is probably better represented as a class, similar to how we refactored the model fitter. Else we are passing around a bunch of state information needlessly, and parameter config is spread out.